CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+### [0.0.8](https://github.com/rhobs/monitoring-stack-operator/commit/) (2022-05-17)
```

Comment:
This link url is the same for every release and points to master. Wouldn't it be better if it linked to the actual tag?

Reply:
Agree ... 
I think this happens because we use `--skip.tag` and let the CI create and push the tag. However, referring to the tag that will be created later should be fine.